integration-tests/README.md

```diff
@@ -0,0 +1,24 @@
+# Integration tests
+
+## Running the Tests
+
+To run the tests, use the following command structure, replacing the placeholders with your actual database details:
+The dbconfig json is a map of postgres versions to details for a db of that version. This corelation will be validated in the tests
+
+```bash
+go test ./... -v -timeout 20m -args -dbconfig='{
+  "16": {
+    "description": "PostgreSQL 16 instance",
+    "db_conn_string": "postgres://username:password@host:port/dbname?sslmode=require",
```

Comment:
`db_conn_string` is largely redundant with host, username, and password. How do we resolve this?